README.md

```diff
@@ -80,27 +81,29 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|                     | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 | Kubernetes 1.8 |
 |---------------------|----------------|----------------|----------------|----------------|----------------|
-| client-go 1.4       | +              | ✓              | -              | -              | -              |
-| client-go 1.5       | +              | +              | -              | -              | -              |
-| client-go 2.0       | +              | +              | ✓              | -              | -              |
-| client-go 3.0       | †              | †              | †              | ✓              | -              |
-| client-go 4.0       | †              | †              | †              | +              | ✓              |
-| client-go HEAD      | †              | †              | †              | +              | +              |
+| client-go 1.4       | ✓              | -              | -              | -              | -              |
+| client-go 1.5       | +              | -              | -              | -              | -              |
+| client-go 2.0       | +-             | ✓              | +-             | +-             | +-             |
+| client-go 3.0       | +-             | +-             | ✓              | -              | +-             |
+| client-go 4.0       | +-             | +-             | +-             | ✓              | +-             |
+| client-go 5.0       | +-             | +-             | +-             | +-             | ✓              |
+| client-go HEAD      | +-             | +-             | +-             | +-             | +              |
 
 Key:
 
 * `✓` Exactly the same features / API objects in both client-go and the Kubernetes
   version.
-* `+` client-go has features or api objects that may not be present in the
-  Kubernetes cluster, but everything they have in common will work. Please
-  note that alpha APIs may vanish or change significantly in a single release.
-* `†` client-go has new features or api objects, and some APIs running in the
-  cluster may have been deprecated and removed from client-go. But everything
-  they share in common (i.e., most APIs) will work.
-* `-` The Kubernetes cluster has features the client-go library can't use
-  (additional API objects, etc).
+* `+` client-go has features or API objects that may not be present in the
+  Kubernetes cluster, either due to that client-go has additional new API, or
+  that the server has deprecated old API. However, everything they have in
```

Comment:
More than deprecated it must have actually been removed.

Reply:
Done.